src/rtc.rs

```diff
@@ -180,7 +179,7 @@ impl Rtc {
       ```
     */
     pub fn wait_alarm(&mut self) -> nb::Result<(), Infallible> {
-        if self.regs.crl.read().alrf().bit() == true {
```

Comment:
It's better use `bit_is_set/bit_is_clear()` here`

Reply:
I agree, didn't think of it; just wanted to do a quick cleanup to shut up clippy. 😅